packages/zent/src/badge/README-en_US.md

```diff
@@ -0,0 +1,162 @@
+## Badge
+
+Badge normally appears in proximity to notification or head picture with eye-catching appeal, typically displaying unread messages count.
+
+### Usage guide
+
+-  Used to prompt for new messages，placed in the upper right corner or right side of the text or icon.
```

Comment:
Badge is used to prompt for new messages，which is placed in the upper right corner or right side of the text or icon